centro-regional-de-estudios-teologicos-de-aragon.csl

```diff
@@ -0,0 +1,443 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" demote-non-dropping-particle="display-and-sort" default-locale="es-ES">
+  <info>
+    <title>Centro Regional de Estudios Teológicos de Aragón (es-ES)</title>
+    <title-short>CRETA</title-short>
+    <id>http://www.zotero.org/styles/centro-regional-de-estudios-teologicos-de-aragon</id>
+    <link href="http://www.zotero.org/styles/centro-regional-de-estudios-teologicos-de-aragon.csl" rel="template"/>
```

Comment:
this link MUST point to a style already on the repository. Add the link to the style you used as a template.